tidb-cloud/connected-care-announcement.md

```diff
@@ -0,0 +1,100 @@
+
+|   Support plan                                            | Basic                        | Developer                                     | Enterprise                                     | Premium                                   |
+|:----------------------------------------------|:-----------------------------|:----------------------------------------------|:-----------------------------------------------|:------------------------------------------|
+| Recommended workloads                          | Personal or starter projects | Workloads under development                   | Workloads under production                     | Business-critical workloads in production |
+| Billing and account inquiries                 | ✔                            | ✔                                             | ✔                                              | ✔                                         |
+| Technical support                             | -                            | ✔                                             | ✔                                              | ✔                                         |
+| Initial response time                         | -                            | Business hours                                | 7x24                                           | 7x24                                      |
+| Connected: Clinic service                     | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: AI chat in IM                      | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM subscription for TiDB Cloud alerts   | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM subscription for ticket updates | -                            | -                                             | ✔                                              | ✔                                         |
+| Connected: IM interaction for support tickets | -                            | -                                             | -                                              | ✔                                         |
+| Technical account manager                     | -                            | -                                             | -                                              | ✔                                         |
+| Pricing                                       | Free                         | Greater of $29 or 10% of total monthly charges | Greater of $699 or 20% of total monthly charges | Contact sales                             |
```

Comment:
Sorry for the late notice. Let's remove the detailed Pricing from the public doc. 
Here we need to delete this row. Thanks.